auth-react/docs/emailpassword/email-password-auth.md

```diff
@@ -32,4 +32,5 @@ __HIGHLIGHT__        return (
        userId: string,
        jwtPayload: any /*JSON object set on the backend*/
    }
-   ```
\ No newline at end of file
+   ```
+- `EmailPasswordAuth` will update the session, because it uses `SessionAuth` underneath. Read more about session updates in [SessionAuth guide](/docs/auth-react/docs/session/session-auth).
```

Comment:
When linking to pages within the same docusaurus project, use relative path

Reply:
I wanted to avoid doing `../` here, but this is only preference. Both styles will cause it to break link if something moves, so if you want me to change to `../`, I will.

Reply:
The reason I insist is that cause in v2 docusaurus (which we are working on moving towards), using relative path doesn't cause a reload of the whole docs..

Reply:
And that's a perfect reason, thanks for explaining! I'll change that.